src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(5)].name();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/41261579-8f15-4000-a4eb-2e2c6b48e1ad)


![image](https://github.com/mate-academy/jv-lottery/assets/20326858/965e2b46-8220-4e83-b3f3-e1ba1cce6aea)

Reply:
not fixed